tests/spec/puppeteer-spec.js

```diff
@@ -0,0 +1,98 @@
+jest.setTimeout(20000);
+
+describe("An editor for MicroPython running at localhost.", function() {
+
+    beforeAll(async() => {
+        global.browser = await global.puppeteer.launch({headless: true, args: ['--no-sandbox', '--disable-setuid-sandbox', '--disable-dev-shm-usage']});
+    });
+
+    afterAll(async() => {
+        global.browser.close();
+    });
+
+    it("Correctly loads a script from an import URL.", async function() {
```

Comment:
Can we also check the script name in this test?

Reply:
There's a future task to check the script name doesn't change when loading an invalid file (i.e. the Makecode test). In the current version of the editor these tests are against, this test will fail.

Reply:
This is for the import URL, do you mean that when loading a script via URL the script name doesn't change?

Reply:
I'm adding script name checks to each test, this was more of a generic comment. No; all other tests change the name correctly.

Reply:
Okay, so are you adding the name checks in this PR? I'm a bit confused as to when that will happen.

Reply:
Changed in https://github.com/microbit-rosslowe/PythonEditor/pull/9/commits/3dcdaf967f1b0b1ec45374b6e936c721b0e0781d